### ISSUE TITLE
DeepLearning_ImageClassification called an error

### ISSUE THREAD

**Issue by batsword** [dotnet/machinelearning-samples#1042]
Tensorflow.TensorflowException:“Received a label value of 6 which is outside the valid range of [0, 2).  Label values: 6 5 5 3 6 0 1 0 2 0
	 [[{{node cross_entropy/sparse_softmax_cross_entropy_loss/xentropy/xentropy}}]]”

can't get pb
